setup-production.adoc

```diff
@@ -31,217 +31,172 @@ $ cd taiga-docker/
+POSTGRES_DB=taiga  # PostgreSQL database name
+POSTGRES_USER=taiga  # user to connect to PostgreSQL
+POSTGRES_PASSWORD=taiga  # database user's password
+POSTGRES_HOST=taiga-db  # Docker service hosting PostgreSQL
+POSTGRES_PORT=5432  # default database port
 ----
 
-**Service: taiga-back**
+.**URLs settings**
+
+These vars set where your Taiga instance should be served, and the security protocols to use in the communication layer.
 [source, bash]
 ----
-POSTGRES_DB: taiga
-POSTGRES_USER: taiga
-POSTGRES_PASSWORD: taiga
+TAIGA_SCHEME=http  # serve Taiga using "http" or "https" (secured) connection
+TAIGA_DOMAIN=localhost:9000  # Taiga's base URL
+SUBPATH=""  # it'll be appended to the TAIGA_DOMAIN (use either "" or a "/subpath")
+WEBSOCKETS_SCHEME=ws  # events connection protocol (use either "ws" or "wss")
 ----
 
-Additionally, you can also configure `POSTGRES_PORT` in `taiga-back`. Defaults to '5432'.
-
-.Taiga Settings
-
-**Service: taiga-back**
-
-The default configuration assumes Taiga is being served in a **subdomain**:
+The default configuration assumes Taiga is being served in a **subdomain**. For example:
 [source, bash]
 ----
-TAIGA_SECRET_KEY: "taiga-back-secret-key"
-TAIGA_SITES_SCHEME: "https"
-TAIGA_SITES_DOMAIN: "taiga.mycompany.com"
-TAIGA_SUBPATH: ""
+TAIGA_SCHEME=https
+TAIGA_DOMAIN=taiga.mycompany.com
+SUBPATH=""
+WEBSOCKETS_SCHEME=wss://taiga.mycompany.com
 ----
 
-If Taiga is being served in a **subpath** instead of a subdomain, the configuration should be something like:
+If Taiga is being served in a **subpath**, instead of a subdomain, the configuration should be something like this:
 [source, bash]
 ----
-TAIGA_SECRET_KEY: "taiga-back-secret-key"
-TAIGA_SITES_SCHEME: "https"
-TAIGA_SITES_DOMAIN: "mycompany.com"
-TAIGA_SUBPATH: "/taiga"
+TAIGA_SCHEME=https
+TAIGA_DOMAIN=mycompany.com
+SUBPATH="/taiga"
+WEBSOCKETS_SCHEME=wss://mycompany.com
 ----
 
-In this case, it's necessary to do some configurations in `taiga-front` as well.
+.**Secret Key settings**
 
-**Service: taiga-front**
+This variable allows you to set the secret key in Taiga, used in the cryptographic signing.
 
-The default configuration assumes Taiga is being served in a **subdomain**:
 [source, bash]
 ----
-TAIGA_URL: "https://taiga.mycompany.com"
-TAIGA_WEBSOCKETS_URL: "wss://taiga.mycompany.com"
-TAIGA_SUBPATH: ""
+SECRET_KEY="taiga-secret-key"  # Please, change it to an unpredictable value!
 ----
 
-If Taiga is being served in a **subpath** instead of a subdomain, the configuration should be something like:
-[source, bash]
-----
-TAIGA_URL: "https://mycompany.com"
-TAIGA_WEBSOCKETS_URL: "wss://mycompany.com"
-TAIGA_SUBPATH: "/taiga"
-----
+.**Email settings**
 
-**Service: taiga-events**
-[source, bash]
-----
-TAIGA_SECRET_KEY: "taiga-back-secret-key"
-----
+By default, emails will be printed in the standard output (`EMAIL_BACKEND=**console**`). If you have a SMTP service, change it to `EMAIL_BACKEND=**smtp**` and configure the rest of these variables:
```

Comment:
As here we tipically find many questions, I'd add something like. "Your SMTP provider will give you the proper values for these settings."

Reply:
I'll leave it this way: "If you own a SMTP service, change it to `EMAIL_BACKEND=**smtp**` and configure the rest of these variables with the proper values supplied by your SMTP provider"